### PR TITLE
dev-db/sqlcipher-3.3.0: add libressl use flag and update HOMEPAGE

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

File: dev-db/sqlcipher/sqlcipher-3.3.0.ebuild [hunk 1]
```diff
@@ -8,17 +8,19 @@ AUTOTOOLS_AUTORECONF=1
...
 KEYWORDS="amd64 x86"
 
-IUSE="readline static-libs tcl test"
+IUSE="readline libressl static-libs tcl test"
 
 # Tcl is always needed by buildsystem
-RDEPEND="dev-libs/openssl:0[${MULTILIB_USEDEP}]
+RDEPEND="
+	!libressl? ( dev-libs/openssl:0[${MULTILIB_USEDEP}] )
+	libressl? ( dev-libs/libressl[${MULTILIB_USEDEP}] )
```
Comment (SoapGentoo) [inline dev-db/sqlcipher/sqlcipher-3.3.0.ebuild]: both openssl and libressl need `:0=`

Comment (soredake) [inline dev-db/sqlcipher/sqlcipher-3.3.0.ebuild]: done

**Comment by monsieurp** [comment]
@SoapGentoo can we move on?

**Comment by soredake** [comment]
ping

**Comment by soredake** [comment]
ping

File: dev-db/sqlcipher/sqlcipher-3.3.0.ebuild [hunk 2]
```diff
@@ -8,17 +8,19 @@ AUTOTOOLS_AUTORECONF=1
...
 KEYWORDS="amd64 x86"
 
-IUSE="readline static-libs tcl test"
+IUSE="readline libressl static-libs tcl test"
 
 # Tcl is always needed by buildsystem
-RDEPEND="dev-libs/openssl:0[${MULTILIB_USEDEP}]
+RDEPEND="
+	!libressl? ( dev-libs/openssl:0[${MULTILIB_USEDEP}] )
+	libressl? ( dev-libs/libressl:0[${MULTILIB_USEDEP}] )
```
Comment (SoapGentoo) [inline dev-db/sqlcipher/sqlcipher-3.3.0.ebuild]: both openssl and libressl need `:0=`

**Comment by SoapGentoo** [comment]
@soredake can you please squash your fix commit into the first one? Have a look at how git rebase is your friend: https://wiki.gentoo.org/wiki/Gentoo_Github#Rebasing_commits

**Comment by soredake** [comment]
Squashed

**Comment by Pinkbyte** [comment]
Closed manually via bd74920e03ccbd498fd8bf228fb198aedfda094f
Thanks!

**Pull request closed without merging**